src/ORM/AssociationCollection.php

```diff
@@ -15,6 +15,7 @@
 namespace Cake\ORM;
 
 use ArrayIterator;
+use Cake\Datasource\AssociationsNormalizerTrait;
```

Comment:
I moved `AssociationsNormalizerTrait` to the datasource package because its not very specific to the ORM.